data/GalaxyStellarMassAlphatoIron/conversion/convertGallazzi2021.py

```diff
@@ -0,0 +1,90 @@
+from velociraptor.observations.objects import ObservationalData
+
+import unyt
+import numpy as np
+import os
+import sys
+
+# Exec the master cosmology file passed as first argument
+with open(sys.argv[1], "r") as handle:
+    exec(handle.read())
+
+# Cosmology
+h_sim = cosmology.h
+h_obs = 0.704  # WMAP7
+Z_solar_obs = 0.02
+
+input_filename = "../raw/Gallazzi_2021_ascii.txt"
```

Comment:
```suggestion
input_filename = "../raw/gallazzi_2021_ascii.txt"
```
As also revealed by the automatic testing: the name used here is not the same as the name of the actual file.

Reply:
Fixed.